client/cypress/e2e/grid-component.cy.ts

```diff
@@ -1,48 +1,75 @@
-import { GridPage } from "cypress/support/grid.po";
+import { GridPage } from 'cypress/support/grid.po';
 
 describe('Grid Component', () => {
-    const page = new GridPage();
+  const page = new GridPage();
 
-    beforeEach(() => {
-      page.navigateTo();
-      // cy.visit('/grid');
-    //   cy.visit('localhost:4200/grid');
-    });
+  beforeEach(() => {
+    page.navigateTo();
+    cy.task('seed:database');
+  });
 
-    before(() => {
-      cy.task('seed:database');
-    })
+  // it('should open saved grid to server', () => {
+  //   cy.get(':nth-child(3) > button').click();
+  //   cy.get(':nth-child(3) > button').click();
+  //   cy.intercept('api/grid').as('saveGrid');
+  //   cy.url().should('match', /\/grid\/67282673702f8c628808e12e$/);
```

Comment:
Test works, just not on owen's specific computer, should be un-comented in soon upcoming pull request for highlighting